### PR TITLE
fix: Attempt to fix update-apk script

### LINKED ISSUE #2794: Fix the update-apk script.
**Describe the chore**
Fix the update apk script.

**Would you like to work on the issue?**

- [X] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by yashk2000**
Fixed #2794 

Changes:  Attempt to fix the update-apk script.

### CODE REVIEW

**Comment by yashk2000** [comment]
@adityastic @iamareebjamal @abishekvashok please review.

File: scripts/update-apk.sh [hunk 1]
```diff
@@ -22,8 +22,10 @@ else
 	/bin/rm -f phimpme-dev-*.apk
 fi
 
-\cp -r ../app/build/outputs/apk/app-debug.apk app-debug.apk
-\cp -r ../app/build/outputs/apk/app-release-unsigned.apk app-release-unsigned.apk
+\cp -r ../app/build/outputs/apk/*/**.apk .
+\cp -r ../app/build/outputs/apk/debug/output.json debug-output.json
+\cp -r ../app/build/outputs/apk/release/output.json release-output.json
+\cp -r ../README.md .
```
Comment (iamareebjamal) [inline scripts/update-apk.sh]: Unneeded

**Comment by yashk2000** [comment]
@iamareebjamal is this fine?

**Comment by yashk2000** [comment]
@iamareebjamal `./gradlew spotlessCheck` is failing. It is saying that some resources are not getting downloaded. But I checked all the links shown in the error, they are all functional.

**Comment by yashk2000** [comment]
@iamareebjamal @abishekvashok please review.

**Comment by adityastic** [comment]
@yashk2000 can you explain to me what are you trying to fix? I'm out of context currently

**Comment by yashk2000** [comment]
@adityastic the update-apk script which updates the apk every time a pr is merged, is not working properly right now. It is showing that `app-debug.apk` does not exist.

I am trying to fix that.

File: scripts/update-apk.sh (hunk 1 above)
Comment (iamareebjamal) [reply scripts/update-apk.sh]: Still not resolved

Comment (yashk2000) [reply scripts/update-apk.sh]: @iamareebjamal what do I need to remove?

Comment (iamareebjamal) [reply scripts/update-apk.sh]: README

Comment (yashk2000) [reply scripts/update-apk.sh]: I purposely left the readme. I thought that the apk branch will look better with the readme. Right now the branch looks dull as there are just a few files. 
I was also planning to add a small section to the readme telling a user that the apk can directly be downloaded from the apk branch.

Comment (yashk2000) [reply scripts/update-apk.sh]: Please tell me if it is alright or if I should remove the readme file.

Comment (iamareebjamal) [reply scripts/update-apk.sh]: remove

Comment (yashk2000) [reply scripts/update-apk.sh]: ok. On it.

**Comment by yashk2000** [comment]
@iamareebjamal please review. Made the change.

**Pull request merged**